tests/test_store.py

```diff
@@ -7,7 +7,8 @@
 from sqlalchemy.exc import IntegrityError, SQLAlchemyError
 import testing.postgresql
 
-from tests.test_data import invalid_message, test_message, updated_message, missing_tx_id_message
+from tests.test_data import invalid_message, test_message, updated_message, missing_tx_id_message,\
```

Comment:
Do multiple import statements rather than line breaks:

```Python
from tests.test_data import missing_tx_id_message, invalid_message
from tests.test_data import test_message, updated_message
```